--- default_postgresql.conf	2011-06-23 22:39:29.000000000 +0530
+++ postgresql.conf	2011-06-11 22:15:41.538808364 +0530
@@ -104,7 +104,7 @@
 
 # - Memory -
 
-shared_buffers = 24MB			# min 128kB
+shared_buffers = 128MB			# min 128kB
 					# (change requires restart)
 #temp_buffers = 8MB			# min 800kB
 #max_prepared_transactions = 0		# zero disables the feature
@@ -114,7 +114,7 @@
 # It is not advisable to set max_prepared_transactions nonzero unless you
 # actively intend to use prepared transactions.
 #work_mem = 1MB				# min 64kB
-#maintenance_work_mem = 16MB		# min 1MB
+maintenance_work_mem = 256MB		# min 1MB
 #max_stack_depth = 2MB			# min 100kB
 
 # - Kernel Resource Usage -
@@ -153,7 +153,7 @@
 #wal_sync_method = fsync		# the default is the first option 
 					# supported by the operating system:
 					#   open_datasync
-					#   fdatasync (default on Linux)
+					#   fdatasync
 					#   fsync
 					#   fsync_writethrough
 					#   open_sync
@@ -167,7 +167,7 @@
 
 # - Checkpoints -
 
-#checkpoint_segments = 3		# in logfile segments, min 1, 16MB each
+checkpoint_segments = 20		# in logfile segments, min 1, 16MB each
 #checkpoint_timeout = 5min		# range 30s-1h
 #checkpoint_completion_target = 0.5	# checkpoint target duration, 0.0 - 1.0
 #checkpoint_warning = 30s		# 0 disables
@@ -384,7 +384,7 @@
 # AUTOVACUUM PARAMETERS
 #------------------------------------------------------------------------------
 
-#autovacuum = on			# Enable autovacuum subprocess?  'on' 
+autovacuum = on			# Enable autovacuum subprocess?  'on' 
 					# requires track_counts to also be on.
 #log_autovacuum_min_duration = -1	# -1 disables, 0 logs all actions and
 					# their durations, > 0 logs only
@@ -446,11 +446,11 @@
 					# encoding
 
 # These settings are initialized by initdb, but they can be changed.
-lc_messages = 'en_IN.UTF-8'			# locale for system error message
+lc_messages = 'en_US.utf8'			# locale for system error message
 					# strings
-lc_monetary = 'en_IN'			# locale for monetary formatting
-lc_numeric = 'en_IN'			# locale for number formatting
-lc_time = 'en_IN'				# locale for time formatting
+lc_monetary = 'en_US.utf8'			# locale for monetary formatting
+lc_numeric = 'en_US.utf8'			# locale for number formatting
+lc_time = 'en_US.utf8'				# locale for time formatting
 
 # default configuration for text search
 default_text_search_config = 'pg_catalog.english'
